dora/src/bytecode/generator.rs

```diff
@@ -2292,6 +2316,10 @@ impl<'a, 'ast> AstBytecodeGen<'a, 'ast> {
                 specialize_type(self.vm, ty, &cls_type_params, type_params)
             }
 
+            CallType::ModuleMethod(_, _, ref type_params) => {
+                specialize_type(self.vm, ty, &TypeList::empty(), type_params)
```

Comment:
I suppose this should look like Method. Could you please add a test for this?

Reply:
If a module can't have type arguments right now, we could assert that the class does not have type parameters.

Reply:
The idea is that modules can never have type arguments, at most their parent class can have them:

```
class Foo[T] { fun foo() -> T = ...
module Bar : Foo[Int] { fun foo() -> Int = ... ]
```